### PR TITLE
figma-landing-page

### PR DESCRIPTION
**Pull request by udhaymanhas**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,8 @@ Juntos is a web application that lets you co watch streaming service such as you
 
 # Artefacts  
 * [Product Requirement Document](/prd/Juntos_PRD.pdf) 
-* [System Design](/system-design/Juntos_system_design.pdf)
\ No newline at end of file
+* [System Design](/system-design/Juntos_system_design.pdf)
+
+# UI Design
```
Comment (chirgjn) [inline README.md:8]: I think this can be a section under "Artefacts"

**Review by chirgjn — CHANGES_REQUESTED** [review]
We need a better name for the "'ui-pages" folder.
I think we can create an "artefacts" folder and keep these pdfs in subfolders under the same.

Can we add a figma link in the PDF. Also checking in the `.fig` would be nice.

File: README.md (hunk 1 above)
Comment (udhaymanhas) [reply README.md]: This is done. 
Cannot add links under pdf, will have to edit them separately to add the link, or will have to modify the screen to add a URL on top. Added Figma link in README instead. 
Created artefacts folder, added a Figma folder and page-name-folder under it.

Comment (chirgjn) [reply README.md]: We can go with the following approach

```md
### UI design
 * Landing page [PDF]() [Figma]()
```

**Review by chirgjn — APPROVED** [review]
The design looks great!!

Please go through the unresolved comments and address the changes

**Comment by chirgjn** [comment]
Instead of `artefacts/landing-page-design/landing-page-design.pdf` can we do `artefacts/ux-design/landing-page-design.pdf`

**Comment by chirgjn** [comment]
@udhaymanhas can you make the final changes so that we can close this one

File: README.md (hunk 1 above)
Comment (udhaymanhas) [reply README.md]: @chirgjn  Now that all designs are complete. 
Do you want me to make any change to this?

File: README.md [hunk 2]
```diff
@@ -3,4 +3,7 @@ Juntos is a web application that lets you co watch streaming service such as you
 
 # Artefacts  
 * [Product Requirement Document](/prd/Juntos_PRD.pdf) 
-* [System Design](/system-design/Juntos_system_design.pdf)
\ No newline at end of file
+* [System Design](/system-design/Juntos_system_design.pdf)
+
+### UI Design
+* Landing page [PDF](/artefacts/landing-page-design/landing-page-design.pdf) [Figma](https://www.figma.com/file/xSAo5U2oFlfKhlZ2dsxnkZ/WireFrame---Juntos?node-id=0%3A1)
```
Comment (chirgjn) [inline README.md:9]: The link to the PDF is broken

**Review by chirgjn — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (udhaymanhas) [reply README.md]: Apologies, this is resolved.

**Review by chirgjn — APPROVED** [review]

**Pull request merged**